tm2py/components/network/transit/transit_assign.py

```diff
@@ -2231,13 +2231,41 @@ def _journey_levels(self) -> EmmeTransitJourneyLevelSpec:
 
             elif self.name == "WLK_TRN_WLK":
                 new_journey_levels = copy.deepcopy(journey_levels)
-                transition_rules = copy.deepcopy(journey_levels[0]["transition_rules"])
+
+                for i in range(0, len(new_journey_levels)):
+                    jls = new_journey_levels[i]
+                    jls["transition_rules"].extend(
+                        [
+                            {"mode": "e", "next_journey_level": i+1},
+                            {"mode": "w", "next_journey_level": i+1},
+                            {
+                                "mode": "a",
+                                "next_journey_level": i+1,
+                            },
+                        ]
+                    )
+                # level 0: only allow walk access and walk auxilary
+                # must use the trasit modes to get onto the next level, 
+                transition_rules_walk = copy.deepcopy(journey_levels[0]["transition_rules"])
+                transition_rules_walk.extend(
+                    [
+                        {
+                            "mode": "e",
```

Comment:
Every level will have all the transit and auxiliary modes. If a value of a mode isn't specified, it will default to the current level, I guess it's still 0. But we can check the run log to confirm.

Reply:
we could also create a new journey level to prohibit `e` in level 0.